includes/class-wc-payment-tokens.php

```diff
@@ -212,4 +213,22 @@ public static function get_token_type_by_id( $token_id ) {
 		$data_store = WC_Data_Store::load( 'payment-token' );
 		return $data_store->get_token_type_by_id( $token_id );
 	}
+
+	/**
+	 * Get classname based on token type.
+	 *
+	 * @since 3.8.0
+	 * @param string $type Token type.
+	 * @return string
+	 */
+	protected static function get_token_classname( $type ) {
```

Comment:
Either this function name needs to change to get_token_class or the calls to self::get_token_class should change to self::get_token_classname